src/ImageRunModal.jsx

```diff
@@ -691,8 +719,16 @@
                     <TextInput id='run-image-dialog-name'
                            className="image-name"
                            placeholder={_("Container name")}
+                           validated={dialogValues.validationFailed.containerName ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230907-093411-a9f37bd4-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)